js/pages/samples.js

```diff
@@ -109,17 +109,19 @@ export const renderSamplesPage = async () => {
                         </div>
                     </div>
                     <div class="row" style="width:100%">
-                        <div class="consentHeadersFont" style="color:#606060;width:100%">
-                            <div>
-                                What Will Happen During My Appointment?
-                            </div>
-                        </div>
-                        <div class="messagesBodyFont" style="width:100%">
-                            <div>
-                                ${site.whatHappens}
-                            </div>
-                        </div>
+                
+                    ${site.name !== "Henry Ford Health" ? `
+                    <div class="consentHeadersFont" style="color:#606060;width:100%">
```

Comment:
I hate to be picky about this, but can we please line up these `div` sections appropriately? If it's just looking weird on my screen, I apologize.

Reply:
@anthonypetersen Made the required changes, please let me know if there is anything else that has to be changed.